endpointmanager/pkg/endpointmanager/postgresql/npiorganizationstore.go

```diff
@@ -192,3 +212,39 @@ func (s *Store) DeleteNPIOrganization(ctx context.Context, org *endpointmanager.
 
 	return err
 }
+
+// GetAllNormalizedOrgNames gets list of all primary and secondary names
+func (s *Store) GetAllNormalizedOrgNames(ctx context.Context) ([]endpointmanager.NPIOrganization, error)  {
+	sqlStatement := `
+	SELECT id, normalized_name, normalized_secondary_name FROM npi_organizations`
+	rows, err := s.DB.QueryContext(ctx, sqlStatement)
+	if err != nil {
+		return nil, err
+	}
+	var orgs []endpointmanager.NPIOrganization
+	defer rows.Close()
+	for rows.Next() {
+		var org endpointmanager.NPIOrganization
+		err = rows.Scan(&org.ID, &org.NormalizedName, &org.NormalizedSecondaryName)
+		if err != nil {
+		  // handle this error
```

Comment:
why panic?

Reply:
Because I forgot to go back and "handle this error". Changed it to return the error instead